apis/nucleus/src/utils/cell-padding.js

```diff
@@ -1,69 +1,68 @@
 import { generateFiltersString } from './generateFiltersInfo';
 
-const NP_PADDINH_IN_CARDS_WITHOUT_TITLE = ['action-button', 'sn-nav-menu'];
+const NO_BORDER_IN_CARDS = ['action-button', 'sn-nav-menu', 'filterpane', 'sn-shape'];
 const NO_PADDING_IN_CARDS = [
+  ...NO_BORDER_IN_CARDS,
   'pivot-table',
   'table',
   'sn-table',
   'sn-pivot-table',
   'kpi',
-  'map',
-  'filterpane',
   'sn-calendar',
   'sn-filter-pane',
   'sn-layout-container',
-  'sn-listbox',
-  'sn-shape',
   'sn-tabbed-container',
 ];
 
-const shouldUseCardPadding = ({ isCardTheme, layout, isError }) => {
-  const visualization = layout?.visualization;
+const getPaddTitle = (visualization) => NO_BORDER_IN_CARDS.indexOf(visualization) === -1;
```

Comment:
not sure if this variation should be kept, and if yes maybe it should also apply to footer?

Reply:
This is the extra horizontal padding for some odd reason? I'd be happy to see if we can get rid of it.

Reply:
in most object we do want the padding to get some space to the border.
but not sure if we also want the padding for object that don't have border (by default)

Reply:
don't know really